File: csharp/RepositoryDemo/Models/Course.cs
using System.Collections.Generic;

namespace RepositoryDemo.Models
{
    public class Course
    {
        public int CourseId { get; set; }
        public string Title { get; set; }
        public int Credits { get; set; }

        public virtual ICollection<Enrollment> Enrollments { get; set; } = new List<Enrollment>();
    }
}

File: springboot/src/main/java/com/example/dp/models/Course.java
package com.example.dp.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.List;

@Entity
@Getter
@Setter
@NoArgsConstructor
@Table(name="courses")
public class Course {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private int courseId;

    @Column(name = "title")
    private String title;

    @Column(name = "credits")
    public int credits;

    @OneToMany(mappedBy = "course")
    public List<Enrollments> enrollmentsList;
}
